ros2controlcli/ros2controlcli/verb/reload_libraries.py

```diff
@@ -30,9 +32,9 @@ def add_arguments(self, parser, cli_name):
         add_controller_mgr_parsers(parser)
 
     def main(self, *, args):
-        response = reload_controller_libraries(args.controller_manager, force_kill=args.force_kill)
-        if response.ok:
-            print('Reload successful')
-        else:
-            print('Error reloading libraries, check controller_manager logs', file=sys.stderr)
-        return not response.ok
+        with NodeStrategy(args) as node:
+            response = reload_controller_libraries(
+                node, args.controller_manager, force_kill=args.force_kill)
+            if response.ok:
+                return 'Reload successful'
```

Comment:
shouldn't we keep the behaviour of stderr prints when something went wrong?

Reply:
the ros2cli infrastructure takes care of this. Any non-zero output will be printed to command line.